projects/cluster/roles/cluster_build_push_image/tasks/main.yml

```diff
@@ -1,43 +1,22 @@
----
 - name: Create the src directory
   file:
     path: "{{ artifact_extra_logs_dir }}/src"
     state: directory
     mode: '0755'
 
 - name: Create namespace for images if does not exist
-  shell:
-    set -o pipefail;
-    oc create namespace "{{ cluster_build_push_image_namespace }}"
-       --dry-run=client
-       -oyaml
-    | tee "{{ artifact_extra_logs_dir }}/src/000_namespace.yml"
-    | oc apply -f-
+  shell: set -o pipefail; oc create namespace "{{ cluster_build_push_image_namespace }}" --dry-run=client -oyaml | tee "{{ artifact_extra_logs_dir }}/src/000_namespace.yml" | oc apply -f-
```

Comment:
@smandaRH , can you try to revert the modifications of this file?  they remove new lines, we don't really want that

Reply:
Sure, will modify the changes requested.